OpenDDSharp/OpenDDSharp/RtpsDiscovery.cpp

```diff
@@ -30,11 +30,14 @@ OpenDDSharp::OpenDDS::RTPS::RtpsDiscovery::RtpsDiscovery(System::String^ name) :
 }
 
 OpenDDSharp::TimeValue OpenDDSharp::OpenDDS::RTPS::RtpsDiscovery::ResendPeriod::get() {
```

Comment:
Changes on `MulticastInst.cpp`, `RtpsDiscovery.cpp` and `RtpsUdpInst.cpp` are not correct and must be reverted.

Instead, we should create a new explicit converter between `OpenDDSharp::TimeValue` and `::OpenDDS::DCPS::TimeDuration` in the `TimeValue.h` file

```cpp
static operator TimeValue(::OpenDDS::DCPS::TimeDuration td) {
    TimeValue t;
    t.Seconds = td.value().sec();
    t.MicroSeconds = td.value().usec();
    return t;
}

static operator ::OpenDDS::DCPS::TimeDuration(TimeValue t) {
    return ::OpenDDS::DCPS::TimeDuration(t.Seconds, t.MicroSeconds);
}
```

In addition, the `RtpsUdpInst->NakDepth` default value seems to be changed in OpenDDS v3.14:

[https://github.com/objectcomputing/OpenDDS/blob/16b426a3ccdba5360b06d9bff625c56dc04e511f/dds/DCPS/transport/rtps_udp/RtpsUdpInst.cpp#L38](https://github.com/objectcomputing/OpenDDS/blob/16b426a3ccdba5360b06d9bff625c56dc04e511f/dds/DCPS/transport/rtps_udp/RtpsUdpInst.cpp#L38
)

And the `MulticastInst.SynInterval` to 30 seconds and 0 msec, please adapt the corresponding unit tests in concordance.